### PR TITLE
Add GASPYSAT.

### PR DESCRIPTION
**Pull request by Phrohdoh**

### CODE REVIEW

File: rules/structures.yaml [hunk 1]
```diff
@@ -371,6 +371,33 @@ GAOREP:
...
+		Footprint: xx xx
+		Dimensions: 2,2
+	Health:
+		HP: 1000
+	Armor:
+		Type: Wood
+	RevealsShroud:
+		Range: 5c0
+	WithIdleOverlay@dish:
+		Sequence: idle-dish
```
Comment (abcdefg30) [inline rules/structures.yaml]: This should define `PauseOnLowPower: true`, shouldn't it?

**Comment by Phrohdoh** [comment]
Updated.

File: rules/structures.yaml (hunk 1 above)
Comment (Phrohdoh) [inline rules/structures.yaml]: Done.

File: rules/structures.yaml [hunk 2]
```diff
@@ -371,6 +371,34 @@ GAOREP:
...
 # TODO: Increase income by 25%
 
+GASPYSAT:
+	Inherits: ^Building
+	Buildable:
+		Queue: Building
+		BuildPaletteOrder: 90
+		Prerequisites: gatech, ~structures.allies
+	Selectable:
+		Bounds: 96, 98, -2, -24
```
Comment (abcdefg30) [inline rules/structures.yaml]: `Bounds: 96, 98, -4, -30`

File: sequences/structures.yaml [hunk 3]
```diff
@@ -247,6 +247,32 @@ gaorep:
...
+		Start: 0
+		Length: 16
+		ShadowStart: 16
+		Tick: 180
+		Offset: 0, -30
+	make: gtspstmk
+		Start: 0
+		Length: 25
+		ShadowStart: 25
+		Tick: 180
```
Comment (abcdefg30) [inline sequences/structures.yaml]: Imho `90` looks far better.

Comment (Phrohdoh) [inline sequences/structures.yaml]: Are the other ticks fine though?

Comment (abcdefg30) [inline sequences/structures.yaml]: I think so.

**Comment by Phrohdoh** [comment]
Updated.

**Comment by abcdefg30** [comment]
:+1:

**Pull request merged**